great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py

```diff
@@ -121,23 +121,29 @@ def _prescriptive_template(
         cls,
         renderer_configuration: RendererConfiguration,
     ) -> RendererConfiguration:
-        add_param_args: List[AddParamArgs] = [
+        add_param_args: AddParamArgs = (
             ("column", RendererValueType.STRING),
             ("value_set", RendererValueType.ARRAY),
             ("parse_strings_as_datetimes", RendererValueType.BOOLEAN),
-        ]
+        )
         for name, param_type in add_param_args:
             renderer_configuration.add_param(name=name, param_type=param_type)
 
         params = renderer_configuration.params
         template_str = ""
 
         if params.value_set:
-            renderer_configuration = cls._add_value_set_params(
-                renderer_configuration=renderer_configuration
+            array_param_name = "value_set"
+            param_prefix = "v__"
+            renderer_configuration = cls._add_array_params(
+                array_param_name=array_param_name,
+                param_prefix=param_prefix,
+                renderer_configuration=renderer_configuration,
```

Comment:
Decided to move away from this pattern having a different class method for each array-type param. There 4 (so far) different versions of `array_param_name` and `param_prefix` used throughout renderers. No, it does not make sense to make them an `Enum` since these can be anything the contributor defines.